packages/components/src/input-control/stories/index.story.tsx

```diff
@@ -82,3 +89,32 @@ WithEdgeLabel.args = {
 	__unstableInputWidth: '20em',
 	labelPosition: 'edge',
 };
+
+export const ShowPassword: StoryFn< typeof InputControl > = ( args ) => {
+	const [ visible, toggleVisible ] = useReducer(
+		( value ) => ! value,
+		false
+	);
+	return (
+		<InputControl
+			type={ visible ? 'text' : 'password' }
+			label="Password"
+			suffix={
+				<InputControlSuffixWrapper>
+					<Tooltip
+						text={ visible ? 'Hide password' : 'Show password' }
+					>
+						<Button
```

Comment:
Might be better to use the `size="small"` variant so the focus rings don't overlap.

Reply:
Good call.

Reply:
I don't know what's the right way to style things in this context, so I went with inline styles. I also originally went with flex + align items center, but then realized flex was enough (not sure why though). Any suggestions welcome!

Reply:
I think it's just the fact the Button became `display: block` because of the flex parent. Which is equivalent to setting `line-height: 0`.

An inline style is perfectly appropriate here 👍